x-pack/plugins/aiops/tsconfig.json

```diff
@@ -34,26 +34,29 @@
     "@kbn/licensing-plugin",
     "@kbn/logging",
     "@kbn/ml-agg-utils",
+    "@kbn/ml-category-validator",
     "@kbn/ml-date-picker",
     "@kbn/ml-error-utils",
     "@kbn/ml-is-defined",
     "@kbn/ml-is-populated-object",
+    "@kbn/ml-kibana-theme",
+    "@kbn/ml-kibana-theme",
```

Comment:
Duplicate entry here.

Reply:
Updated in https://github.com/elastic/kibana/pull/162319/commits/914c43d19d45dbc61c567f791ec0b4562a8012cf